recipe/build_osx_arm64.sh

```diff
@@ -16,6 +16,12 @@ export CMAKE_PREFIX_PATH=${PREFIX}
 # Filter CMAKE_PREFIX_PATH out of CMAKE_ARGS because scikit-build needs to set it
 export SKBUILD_CONFIGURE_OPTIONS=$(echo $CMAKE_ARGS | sed -e 's/\-DCMAKE_INSTALL_PREFIX\=[^ ]* //g')
 
+# 10.14 required for some C++17 features used in TBB
```

Comment:
Is this right? I thought we moved to C++17 for `std::cyl_bessel_j`, not TBB.

Reply:
We originally did it only for `std::cyl_bessel_j`, yes, but once we raised the C++ standard, there were some internal parts of tbb trying to use the newer standard and required an update to the target platform version to support it.

Reply:
Nice to know! Thanks.